### PR TITLE
Minor Map Fixes 3- The Revengenancing

### LINKED ISSUE #50081: Meta's virology scubber injector is not able to be interacted by borgs or AI
Issue reported from  Round ID: [132546](https://scrubby.melonmesa.com/round/132546)(actually the one before) (/tg/Station Manuel [ENGLISH] [US-WEST] [ROLEPLAY] [99% LAG FREE])

Reporting client version: 513.1514

The injector appeared off and i couldnt get the UI to appear.

### LINKED ISSUE #50256: Camera and light placed in worst possible spot on puby
obraz
They are being hit by emitter.

### LINKED ISSUE #51692: The default robot patrol path is broken in Metastation
## Reproduction: 
1. Make sure you're on Metastation
2. Either build a robot or just find one bumbling around.
3. Set the "Patrol Station" setting to YES
4. Follow the robot around and see how the stupid thing, after a while, keeps coming back to the same places over and over again

Video showing the bug:
https://youtu.be/E7DuRd6X2XE

### LINKED ISSUE #51851: Metastation aft primary hallway air scrubber #2 not connected to pipenet
Issue reported from  Round ID: [140412](https://scrubby.melonmesa.com/round/140412) (/tg/Station Manuel [ENGLISH] [US-WEST] [ROLEPLAY] [99% LAG FREE])

Reporting client version: 513.1507

I've seen this enough to determine it roundstarts in this configuration.
scrubber glitch
I'm not sure if this was passed up in a map overhaul or if it's deliberate to make busywork for those engineer types.

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
Another batch of minor map fixes.
Fixes #51851.
Fixes #50256.
Fixes #51692 (thank you TheSmallBlue).
Fixes #50081.
Fixes the Nanite Lab APC on both box and icebox to not be floating, and the Medbay Lobby APC to not be in a window. Also actually connects the TComms on both box and icebox to the power grid, and adds an APC to Icebox TComms control (really, guys?).
Fixes an incorrectly placed fire alarm on Delta's departures hallway, as well as a rogue pipe.
Fixes the Chemistry Factory APC on Meta floating off the wall.
Integrates changes from #51749, thank you @KathyRyals.

## Why It's Good For The Game
Less map problems, less pain.

## Changelog
:cl: Inept, KathyRyals
fix: A few minor map fixes, hot from the oven.
/:cl:

### CODE REVIEW

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -19949,8 +19950,8 @@
 "bgP" = (
 /obj/machinery/disposal/bin,
 /obj/effect/turf_decal/trimline/yellow/end{
-	icon_state = "trimline_end";
-	dir = 8
+	dir = 8;
+	icon_state = "trimline_end"
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm:19954]: none of these should be here.

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 2]
```diff
@@ -21241,12 +21242,12 @@
 /area/medical/medbay/lobby)
 "bjT" = (
 /obj/effect/turf_decal/trimline/blue/corner{
-	icon_state = "trimline_corner";
-	dir = 1
+	dir = 1;
+	icon_state = "trimline_corner"
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm:21246]: same

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 3]
```diff
@@ -22684,12 +22686,12 @@
 /area/medical/pharmacy)
 "boe" = (
 /obj/effect/turf_decal/trimline/blue/filled/corner{
-	icon_state = "trimline_corner_fill";
-	dir = 1
+	dir = 1;
+	icon_state = "trimline_corner_fill"
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm:22690]: same

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 4]
```diff
@@ -34108,8 +34102,8 @@
 	},
 /obj/item/clothing/glasses/science,
 /obj/effect/turf_decal/trimline/green/filled/line{
-	icon_state = "trimline_fill";
-	dir = 8
+	dir = 8;
+	icon_state = "trimline_fill"
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm:34106]: same

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm [hunk 5]
```diff
@@ -27416,12 +27412,12 @@
 /area/hallway/primary/central)
 "bmV" = (
 /obj/effect/turf_decal/trimline/blue/filled/corner{
-	icon_state = "trimline_corner_fill";
-	dir = 4
+	dir = 4;
+	icon_state = "trimline_corner_fill"
```
Comment (ShizCalev) [inline _maps/map_files/IceBoxStation/IceBoxStation.dmm:27416]: .

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm [hunk 6]
```diff
@@ -28648,8 +28644,8 @@
 /area/science/genetics)
 "bpF" = (
 /obj/effect/turf_decal/trimline/yellow/filled/line{
-	icon_state = "trimline_fill";
-	dir = 1
+	dir = 1;
+	icon_state = "trimline_fill"
```
Comment (ShizCalev) [inline _maps/map_files/IceBoxStation/IceBoxStation.dmm:28648]: .

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm [hunk 7]
```diff
@@ -45507,8 +45524,8 @@
 	dir = 8
 	},
 /obj/structure/chair/sofa/left{
-	icon_state = "sofaend_left";
-	dir = 1
+	dir = 1;
+	icon_state = "sofaend_left"
```
Comment (ShizCalev) [inline _maps/map_files/IceBoxStation/IceBoxStation.dmm:45528]: .

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm [hunk 8]
```diff
@@ -58087,8 +58120,8 @@
 	dir = 8
 	},
 /obj/structure/chair/sofa/right{
-	icon_state = "sofaend_right";
-	dir = 1
+	dir = 1;
+	icon_state = "sofaend_right"
```
Comment (ShizCalev) [inline _maps/map_files/IceBoxStation/IceBoxStation.dmm:58124]: .

**Comment by EOBGames** [comment]
I think that should be the errant variables removed.

**Pull request merged**